.github/workflows/ci.yml

```
@@ -18,13 +18,6 @@ jobs:
         uses: actions/setup-node@v2.5.1
         with:
           node-version: ${{ matrix.node-version }}
```

```suggestion
          node-version: ${{ matrix.node-version }}
          cache: "npm"
```

Don't forget to specify the cache when removing the unnecessary action! ;)

---

Oops, good catch